package.json

```diff
@@ -5,14 +5,20 @@
   "repository": "git@github.com:superfaceai/station.git",
   "author": "Superface Team",
   "scripts": {
+    "build_reporter": "tsc src/reporter/*.ts --outDir dist",
```

Comment:
note: I would give a try to https://github.com/facebook/jest/issues/10105#issuecomment-678600189

Reply:
It works 🎉